tests/pass/intrinsics-x86-avx.rs

```diff
@@ -0,0 +1,162 @@
+// Ignore everything except x86 and x86_64
+// Any additional target are added to CI should be ignored here
```

Comment:
```suggestion
// Any new targets that are added to CI should be ignored here.
```

Reply:
If you could fix this in all test files while you are at it, that would be great. :)